lib/membrane_rtc_engine/endpoints/webrtc/rtp_munger.ex

```diff
@@ -105,7 +106,7 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPMunger do
           extensions: [],
           csrcs: [],
           payload_type: track.payload_type,
-          marker: false,
+          marker: force_marker?,
```

Comment:
Question: what would happen if we always set this to `true`? Wouldn't that prevent browsers from treating padding packets as required for the frame and awaiting retransmission in case it's lost?

Reply:
Honestly speaking I have no idea but we can test/investigate this